man/systemd.service.xml

```diff
@@ -77,6 +77,19 @@
     with SysV</ulink> document.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Service Templates</title>
+
+    <para>It is possible for Systemd services to take a single argument via the
+    <literal><replaceable>service</replaceable>@<replaceable>argument</replaceable></literal>
+    syntax. Such services are called "instantiated" or "template" services. An
```

Comment:
Hmm, so I'd be careful with wording here... I think "foo@bar.service" is an instantiated service name, while "foo@.service" is a template service file name... i.e. I think "instantiated" should refer to something where the instance name is filled in, while "template" should refer to something where it isn't...